pkg/cloud/azure/azure_storage.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/service"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/cockroachdb/cockroach/pkg/base"
+	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl"
```

Comment:
I might be wrong but this feels like a cyclic dependency waiting to happen. Is there a better place we could move this flag?

Reply:
Nothing obvious I can think of, though maybe David has an idea. We could solve it when the problem comes up? But if we also aspire to get rid of this eventually it may not matter.

Reply:
Yeah, no, this dep feels pretty wrong to me too. I'd move `EnableCloudBillingAccounting` to a generic place that both cloud/ and cdc/ can depend on. Maybe util/metric?